lib/std/private/strutils2.nim

```diff
@@ -0,0 +1,55 @@
+##[
```

Comment:
This should be ``compiler/strutils2.nim``, ``lib/std/private`` creates bootstrapping problems and is a contradiction. ("A standard hidden library?")

Reply:
done but see below.